### DISCUSSION TITLE
[Feature Request] Allowing multiple of the same layout with different configs/identifiers

### DISCUSSION THREAD

**Discussion by tokisuno** [qtile/qtile#4806] [Ideas] [answered]
It would be really nice to able to have multiple of the same layout, but only under the exception that they have different IDs so they don't get mixed up. 

The idea would be qtile returns an error if that condition isn't met.

For example, I really like using spiral, but having the option where I could have one spiral with no gaps and another with gaps would be nice. 

Maybe there's something out there already that I don't know about? Like a keymapping of some sort which allows you to change layout properties?

Would love to hear back!

**Comment by elParaguayo** [comment, marked as answer]
You can do this. Just add a different `name ` parameter when you configure the layout.
